components/staging/staging.less

```diff
@@ -13,6 +13,10 @@
     }
   }
 
+  textarea.form-control {
```

Comment:
Would this change all textareas?

Would it be better to add a `class` like `commit-body` and use that?

Reply:
Well, it would change all textareas with the class `form-control` in the `.staging` area...  I don't think there are others?